tools/mkimage-rpi/build.yml

```diff
@@ -1,4 +1,4 @@
-image: mkimage-rpi3
+image: mkimage-rpi4
```

Comment:
ideally the package/image name is the same as the directory name. by renaming it to `mkimage-rpi4` it implies it won't work on rpi3 anymore

Reply:
I will rename this to mkimage-rpi